.github/workflows/release.yml

```
@@ -1,80 +1,16 @@
-name: Release Buildpack
+name: Release Buildpacks
```

(there is just the one buildpack for now)

```suggestion
name: Release Buildpack
```